src/apps/fun.py

```diff
@@ -0,0 +1,30 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+"""
+
```

Comment:
why new line?

Reply:
Because it looks better.

Reply:
Ah, makes perfect sense. LGTM, merge approved